packages/components/src/FormField/FormFieldDescription.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Text } from "../Text";
+
+interface FormFieldDescriptionProps {
+  description: string;
```

Comment:
Should this be readonly?

Reply:
Sure, doesn't change much as it's only used internally, but to be consistent with our other type declarations, i've updated it in 43f730f

Reply:
woot woot!